hdf5.spec

```
@@ -1,29 +1,27 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 %{!?_fmoddir:%global _fmoddir %{_libdir}/gfortran/modules}
 
-%global daos_major 0
-%global hdf5_commit fa40c6c59af5d9aabd4b478cd02f8a9f7ebf7922
-%define hdf5_sha .gfa40c6c59a
+%global daos_major 1
 
 # Patch version?
 %global snaprel %{nil}
-# HDF5 versions
-%global major 1.12
-%global minor 0
+%global hdf5_major 1
+%global hdf5_minor 13_0
```

This is not a valid minor version format.  This looks more like _minor.bugfix_.  This will be incompatible with previously released packages which build their version as such https://github.com/daos-stack/hdf5/blob/651c0a7caf896b1efe6c18bea75bc275dd4bf9b5/hdf5.spec#L11-L18 which yielded packages such as:

hdf5-1.12.0-5.gfa40c6c59a.el7.x86_64.rpm

```suggestion
%global hdf5_minor 13.0
```

Or perhaps even better:

```suggestion
%global hdf5_minor 13
%global hdf5_bugfix 0
```